cabby/model/text/dual_encoder/train.py

```diff
@@ -99,6 +101,7 @@ def evaluate(self, validation_set: bool = True):
           batch_dim, cell_dim, output_dim)
         text_embedding_exp = text_embedding.unsqueeze(1)
         output = self.cos(cellid_embedding_exp, text_embedding_exp)
+        # output = output * distibution
```

Comment:
Remove commented line.

Reply:
Or, perhaps this should have an if statement that multplies by the distance distribution if desired?

Reply:
Added a condition statement